.editorconfig

```diff
@@ -27,6 +27,10 @@ ij_any_add_a_space_at_line_comment_start = true
 indent_style = tab
 indent_size = tab
 
+[*.sol]
+indent_style = space
+indent_size = 4
+
```

Comment:
I don't think this is related to the changes in the staking contracts

Reply:
It isn't, but my IDE was reformatting .sol files in the project since it was following convention of the monorepo which is different from our package. Needed to explicitly state it